runtime/browser/android/cookie_manager.cc

```diff
@@ -7,28 +7,43 @@
 #include <string>
 
 #include "android_webview/browser/scoped_allow_wait_for_legacy_web_view_api.h"
```

Comment:
As we discussed, `ScopedAllowWait` was a private class and `ScopedAllowWaitForLegacyWebViewApi` was declared as friend class. But `ScopedAllowWaitForLegacyWebViewApi` was not specified in which file.
So we can copy `scoped_allow_wait_for_legacy_web_view_api.h` to Crosswalk, remain the class name `ScopedAllowWaitForLegacyWebViewApi`.